### DISCUSSION TITLE
Support for WebGPU

### DISCUSSION THREAD

**Discussion by lmarzocchetti** [odin-lang/Odin#3454] [Ideas/Requests]
Looking at the future of game/graphics programming, i see WebGPU, both as an alternative to WebGL and for a shared layer with various backend ( vulkan, opengl, directx, ecc). Is planned to add as a "vendor" library, some bindings to the Dawn / wgpu implementation for WebGPU, or is it too early for this kind of stuff because the API is still rapidly evolving and changing?

**Comment by karl-zylinski** [comment]
It exists now https://github.com/odin-lang/Odin/tree/master/vendor/wgpu 🎉

**Comment by lmarzocchetti** [comment]
Wow, thank you really. I love Odin community

**Reply by karl-zylinski** [reply]
Thanks go to @laytan who did a lot of the wpgu work!
